src/sentry/rules/actions/sentry_apps/base.py

```diff
@@ -10,6 +10,11 @@
 class SentryAppEventAction(EventAction, abc.ABC):
     """Abstract class to ensure that actions in SENTRY_APP_ACTIONS have all required methods"""
 
+    @property
+    @abc.abstractmethod
+    def actionType(self) -> str:
```

Comment:
Whacky way to force a protocol and allow mypy to help us.  Used for the extracted RPC serializable interface for sending SentryAppEvent objects.